managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+![Embedded Workflow UI](../images/embedworkflow_ui.png){: caption="Embedded Workflow UI" caption-side="bottom"}
+
+- **Workflows**
+
+   This UI page includes a summary of the available workflows. You can use different views to review the workflows:
+
+   - Text view, which is the raw representation (JSON or YAML) of the Amazon State Languages (ASL) code.
+   - Graph view, which shows an SVG visual representation of the workflow structure.
+
+- **Repositories**
+
+   This page lists, and provides a summary of, the added embedded workflow repositories. From this page, you can add, update, and remove repositories.
+   When a repository is added and synchronized, any workflows that are found in the repository can be viewed on the **Workflows** UI page.
+
+- **Credentials**
+
+   Credentials are used by {{ site.data.product.title_short }} for authentication when running embedded workflows against machines, synchronizing with inventory sources, and importing project content from a version control system.
+
+   This page lists, and provides a summary of, the credentials for accessing and using embedded workflows. From this page, you can add, update, and delete credentials.
+
+   These credentials are tied to the provider instance, and are used by the workflow when it runs. The credential types are defined by the provider.
+
+#### UI Field Updates
+{: #field_updates}
+
+Existing integration points are updated for the service catalog creation and management process to support choosing to use **Embedded Workflows** or **Embedded Automate**. This choice can be set when working with service catalog items and service dialogs.
+
+* Service Dialog
+
+   When creating a service dialog, fields that are dynamic now include the option for using **Embedded Workflows** or **Embedded Automate**.
+
+   ![Service Dialog options](../images/embedworkflow_servicedialogdynamic.png){: caption="Service Dialog options" caption-side="bottom"}
+
+* Service Catalog Items
+
+   The provisioning, configuring, or retiring entry points now include the option for using **Embedded Workflows** or **Embedded Automate**.
+
+   ![Entry Points options](../images/embedworkflow_entrypointsembedded.png){: caption="Entry Points options" caption-side="bottom"}
+
+### Adding an Embedded Workflows Repository
+
+As a starting point to begin authoring a workflow, you must first add the repository that includes your workflows.
+
+To add a repository, complete the following steps:
+
+1. Click  **Automation** > **Embedded Workflows** > **Repositories** > **Configuration** > **Add New Repository**.
+
+   ![Add a repository](../images/embedworkflow_addnewrepository.png){: caption="Add a repository" caption-side="bottom"}
+
+2. Enter *Name*, *Description*, and a *GitHub URL*.
+
+3. Provide a Repository Name in the **Name** field.
+
+4. Add a description for the repository in the **Description** field.
+
+5. Add a **GitHub URL** for the repository.
```

Comment:
Are we adding the URL twice?

Reply:
I have updated the procedure to remove the duplicated step.